src/main/resources/reference.conf

```diff
@@ -38,4 +38,11 @@ videobridge {
       }
     }
   }
+  websockets {
+    enabled=false
+    # Must be set when enabled = true
+    # server-id="default-id"
+    # domain="some-domain"
+    # tls=true
```

Comment:
This is an interesting case. The original code handles the case of a missing value gracefully by following the jetty configuration. This doesn't fit with the new idea of requiring values for all properties. We could:
1. Modify `useTls` to return `Boolean?`, catch the exception and return `null` if the property is not defined. 
2. Have a special value (e.g. the string `default`) in `reference.conf`.

I think 1 is cleaner, even though it requires catching an exception.

Reply:
Ah yes, I forgot to take into account the passed value of `tls`.  I'll go with option 1 here.

Reply:
It does feel like this is a legitimate use case for an 'optional' property.  I had originally done this by wiring 'optional' into the core code, but we could also do it with a layer on "top" here: by swallowing the exception and returning null.  Since `ConfigProperty` requires a non-nullable type on the generic, it'd be on top of the config code entirely.  I think I'll do the 'on top' version here, and we can see how much more this comes up and maybe put it back in the core config code.